src/WaCore.Crud.ListSample1/Data/UnitOfWork.cs

```diff
@@ -14,11 +14,15 @@ public UnitOfWork(LibraryDbContext dbContext, IServiceProvider serviceProvider)
         { }
 
         public IBooksListRepository BooksRepository => GetRepository<IBooksListRepository>();
+
+        public ICarRepository CarRepository => GetRepository<ICarRepository>();
```

Comment:
For consistency you should call it CarsRepository or rename the other one to BookRepository

Reply:
Renamed the IBookListRepository

Reply:
But the property in UnitOfWork is still called BooksRepository

Reply:
renamed as well